### PR TITLE
[ja translation] tutorials/* files

### PR DESCRIPTION
**Pull request by uraway**
ref: #220 

files:

> * [ ]  [`Babel with TypeScript.md`](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/Babel%20with%20TypeScript.md)
> * [ ]  [`DOM Manipulation.md`](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/DOM%20Manipulation.md)
> * [ ]  [`React.md`](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/React.md)
> * [ ]  [`TypeScript Tooling in 5 minutes.md`](https://github.com/microsoft/TypeScript-Website/blob/v2/packages/documentation/copy/en/tutorials/TypeScript%20Tooling%20in%205%20minutes.md)

diff: 81e6949

### CODE REVIEW

**Comment by uraway** [comment]
@orta oh, sorry, I didn't notice the localization steps have been changing.

Should I close this PR and create a new one for the TypeScript-Website-Localizations repo, following [this](https://github.com/microsoft/TypeScript-Website-Localizations/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R23)?

**Comment by orta** [comment]
It's OK, I'm switching at the end of the week - I was pinging _old_ PRs - if it gets in before friday, that's fine

File: packages/documentation/copy/ja/tutorials/Babel with TypeScript.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+permalink: /ja/docs/handbook/babel-with-typescript.html
+oneline: BabelとTypeScriptを組み合わせたプロジェクトの作成方法
+translatable: true
+---
+
+## BabelとTypeScriptの`tsc`の比較
+
+モダンなJavaScriptプロジェクトを作る際、TypeScriptからJavaScriptにファイルをトランスパイルするにはどのような方法が正しいのでしょうか？
+
+その答えは、プロジェクトによって異なりますが _"状況次第"_ だったり、 _"誰かが決めてくれたもの"_ であることが多いです。[tsdx](https://tsdx.io)、[Angular](https://angular.io/)、[NestJS](https://nestjs.com/)といった既存のフレームワークや、あるいは[Getting Started](/docs/home)で紹介したようなフレームワークを使ってプロジェクトを構築しているのならば、あなたに代わってこの決定を行ってくれます。
```
Comment (sasurau4) [inline packages/documentation/copy/ja/tutorials/Babel with TypeScript.md]:
```suggestion
その答えは、プロジェクトによって _"状況次第"_ だったり、 _"誰かが決めてくれたもの"_ であることが多いです。[tsdx](https://tsdx.io)、[Angular](https://angular.io/)、[NestJS](https://nestjs.com/)といった既存のフレームワークや、あるいは[Getting Started](/docs/home)で紹介したようなフレームワークを使ってプロジェクトを構築しているのならば、あなたに代わってこの決定を行ってくれます。
```

**Comment by orta** [comment]
Ah dang, I was meant to merge this last night but forgot!

Sorry @uraway - I'm going to close this PR, I'd love to see this get re-created on the new repo though!

https://github.com/microsoft/TypeScript-Website-Localizations

**Pull request closed without merging**